app/src/main/res/layout/line_item.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this layout going to be interesting to look at?

Reply:
As of currently, no - I hadn't realized that wasn't part of this story when I added it. It will look better in the next one though - or would you prefer I remove it from this PR?